core/test/utils/fixtures/export/lts/export-003-mu.json

```diff
@@ -131,7 +131,7 @@
                 "created_at": 1388319501897,
                 "created_by": 1,
                 "updated_at": null,
-                "updated_by": null
+                "updated_by": 10
```

Comment:
We already had a null case, added a different case for this file. (unknown author reference)